File: level_1/PYTHON/이상한 문자 만들기.py
def solution(s):
    answer = ''
    flag =True
    for i in s:
        if i== ' ':
            answer+=' '
            flag=True
            
        elif flag == True:
            answer+=i.upper()
            flag=False
            
        else:
            answer+=i.lower()
            flag=True
        
    return answer

File: level_1/C++/이상한 문자 만들기.cpp
using namespace std;

int solution(int n) {
	int answer = 0;
	if (n == 0) {
		return 0;
	}
	else if (n == 1) {
		return 1;
	}

	for (int i = 1; i<(n / 2) + 1; i++) {
		if (n%i == 0) {
			answer += i;
		}
	}
	return answer + n;
}